packages/lesswrong/components/editor/EditorUnmanaged.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import PropTypes from 'prop-types';
+
+const styles = (theme: ThemeType): JssStyles => ({
```

Comment:
avoid returning `JssStyles`, so that `classes` can be typed `classes: ClassesType<typeof styles>` (even though there aren't any styles in this case)

Reply:
Oops, this file wasn't supposed to be in this commit at all (it was hanging out in my working directory from an unrelated thing)